frontend/src/components/guideModal/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import { Modal, Box } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import CloseButton from '../closeButton'
+import InputTabs, { TabPanelTitle } from '../inputTabs'
+import GuideCarousel from '../guideCarousel'
+
+const GuideModal = observer(({open, onClose}) => {
+  const theme = useTheme()
+
+  return (
+    <Modal
+      open={open}
+      onClose={onClose}
+      sx={{
+        background: 'rgba(242, 244, 252, 0.2)',
+        backdropFilter: 'blur(5px)',
+      }}
+    >
+      <Box sx={{
+        position: 'absolute',
+        right: 0,
+        top: 0,
+        width: '712px',
+        height: '100%',
+        backgroundColor: theme.palette.background.guideModal,
+        padding: '34px',
+      }}>
+        <CloseButton title='Close Guide' onClick={onClose}/>
+        <InputTabs
+          tabStyles={{position: 'absolute', left: '48px', top: '30px'}}
+          renderKeyboardContent={() => (
+            <>
+              <TabPanelTitle>Keyboard Guide</TabPanelTitle>
+              <GuideCarousel />
```

Comment:
I guess the component will be the same for joystick? The only diff will be in a content, right? Do you want to design this component with that knowledge in mind?

Reply:
for now (without having design for joystick) I assume that both guides will use similar components like carousel and it's already separated. Let's keep it as it is for now.